test-target/limit-range.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: LimitRange
+metadata:
+  name: test-limit-range
+spec:
+  limits:
+    - default:
+        memory: 512Mi
+      defaultRequest:
+        memory: 256Mi
+      type: Container
```

Comment:
Shouldn't this be per pod ?

Reply:
Will verify.

Reply:
Okay so the way I understand it is that you can either have a container LR, pod LR, or both applied.  I think as long as you have _something_ applied to the namespace it should cover the test case.

Reply:
I removed the LimitRange as a "default" applied YAML.  I left it in there as an available YAML to apply but since the test in https://github.com/test-network-function/cnf-certification-test/pull/386 is only looking for ResourceQuota's right now it shouldn't be applied by default.